docs/input.md

```diff
@@ -2,32 +2,66 @@
 
 ## Overview
 
-Step 1: Generate the required input files (acoustic two-way travel times,
-surface platform positions, sound velocity profile)
+GNATSS has two processing modes, a "posfilter" mode and a "solver" mode, which
+are used at different phases of the processing chain. It is possible to run both
+processing modes in sequence or individually. The posfilter module outputs
+GNSS-Acosutic travel times and transducer positions in the community standard
+GNSS-Acoustic data format, so if you have obtained data already in that format
+you may skip the posfilter mode and immediately run the solver.
 
-Step 2: Create working directory and config.yaml file
+The basic step-by-step guide to position a GNSS-Acoustic array using GNATSS is:
 
-Step 3: From working directory, execute
+Step 1: Gather the required input files
+
+Step 2: Create working directory and configuration file
```

Comment:
Maybe add link to the configuration file page/section for the words `configuration file`

Reply:
Done.